auto_cpufreq/core.py

```diff
@@ -267,13 +267,23 @@ def turbo(value: bool = None):
     """
     p_state = Path("/sys/devices/system/cpu/intel_pstate/no_turbo")
     cpufreq = Path("/sys/devices/system/cpu/cpufreq/boost")
+    amd_pstate = Path("/sys/devices/system/cpu/amd_pstate/status")
+    scaling_driver = Path("/sys/devices/system/cpu/cpu0/cpufreq/scaling_driver")
 
     if p_state.exists():
         inverse = True
         f = p_state
     elif cpufreq.exists():
         f = cpufreq
         inverse = False
+    elif amd_pstate.exists() and scaling_driver.exists():
+        amd_value = amd_pstate.read_text().strip()
+        scaling_driver_value = scaling_driver.read_text().strip()
+        if amd_value == "active" and scaling_driver_value == "amd-pstate-epp":
```

Comment:
remove the `scaling_driver` check since if `amd_pstate/status` equals "active", then the loaded driver must be `amd-pstate-epp`

Reply:
also this isn't really important but I think this entire if statement doesn't actually do anything since the `cpufreq` Path only doesn't exist on `amd-pstate-epp` but I think we should leave the if statement since it makes it more clear what is actually happening.

Reply:
hmm, by the way, what for amd_pstate == guided or amd_pstate == passive?

Reply:
Self-reply for guided: https://github.com/AdnanHodzic/auto-cpufreq/issues/602#issuecomment-1945918261
/sys/devices/system/cpu/cpufreq/boost  is set.